client/src/components/CreatePost/Form/AskHelp.js

```diff
@@ -3,15 +3,24 @@ import First from "./FirstSection";
 import Second from "./SecondSection";
 import Third from "./ThirdSection";
 import { Footer, Submit } from "components/CreatePost/StyledModal";
+import createPostSettings from "assets/data/createPostSettings";
+
+const { helpTypes } = createPostSettings;
```

Comment:
Uhhh Ohh :O this file was deleted recently....

Reply:
Thanks!!